src/widget/paintable.cpp

```diff
@@ -263,7 +263,7 @@ QString Paintable::getAltFileName(const QString& fileName) {
         return fileName;
     }
 
-    QString newFileName = temp[0] + QString::fromAscii("@2x.") + temp[1];
+    QString newFileName = temp[0] + QString::fromLatin1("@2x.") + temp[1];
```

Comment:
Omit the QString::fromLatin1() wrapper.

Reply:
I switched this to QLatin1String(.) because I think this is the advice of 
https://woboq.com/blog/qstringliteral.html
since it is being immediately passed to QString::operator+